packages/storybook-html/config/preview.tsx

```diff
@@ -37,14 +52,15 @@ const preview: Preview = {
       return (
         <div
           className={clsx('utrecht-document', 'utrecht-document--surface', {
-            'utrecht-reduced-motion': ['1', 'true'].includes(process.env['STORYBOOK_REDUCED_MOTION'] ?? ''),
+            'utrecht-reduced-motion': storyContext.globals?.reducedMotion || isReducedMotion,
```

Comment:
`process.env['STORYBOOK_REDUCED_MOTION']` checken is nog steeds nodig, die stellen we in via GitHub Actions om te voorkomen dat screenshots in Chromatic halverwege een fade-in animatie worden gemaakt bij de Backdrop component. Zou je die instelling weer willen herstellen, en gelijk ook een comment plaatsen zodat duidelijk is waar het voor bedoeld is, en om te voorkomen dat het in de toekomst toch per ongeluk zou verdwijnen?

`matchMedia` lijkt me niet nodig om te gebruiken, want dat gedeelte kan gewoon via CSS.